app/src/main/java/com/mho/training/sources/CreditDataSource.kt

```diff
@@ -15,24 +15,24 @@ import java.io.IOException
 
 class CreditDataSource(private val resources: Resources) : RemoteCreditDataSource {
```

Comment:
You could receive the error message string instead of receive the resources object

Reply:
Got it!